mitodl/odl-video-service
# Dorska/anonymous tos

Wait for #581 and #584 to land before reviewing this.

#### What are the relevant tickets?
Fixes #583.

#### What's this PR do?
1. Allows anonymous access to the terms-of-service page.

#### How should this be manually tested?
1. For user in (anonymous, logged_in):
   a. ensure that you can view the terms-of-service page.
   b. ensure that the drawer looks reasonable on that page.
